src/Components/DiaryCard.js

```diff
@@ -27,35 +27,22 @@ function DiaryCard({title, subtitle, description, color }) {
     const resultString = more ? description.slice(0, 100) + '...' : description;
 
     function ShowMore() {
```

Comment:
For the function declaration in react, start with simple character. ( I missed this in previous review)